装了python 3.0后会出现运行HTMLTestrunner报错，因为里面的脚本还是用的2.7版本上的

语法。现在将需要修改的地方总结如下：

打开你的HTMLTestruner.py文件：

第94行，将import StringIO修改成import io

第94行，将import StringIO修改成import io

第539行，将self.outputBuffer = StringIO.StringIO()修改成self.outputBuffer=io.StringIO()

第539行，将self.outputBuffer = StringIO.StringIO()修改成self.outputBuffer=io.StringIO()

第774行，将uo = o.decode(‘latin-1‘)修改成uo =e

第774行，将uo = o.decode(‘latin-1‘)修改成uo =e

第775行，将ue = e.decode(‘latin-1‘)修改成ue = e

第775行，将ue = e.decode(‘latin-1‘)修改成ue = e

第636，将print >> sys.stderr, ‘\nTime Elapsed: %s‘%(self.stopTime-self.startTime)修改成print(sys.stderr, ‘\nTimeElapsed: %s‘ % (self.stopTime-self.startTime))

第636，将print >> sys.stderr, ‘\nTime Elapsed: %s‘%(self.stopTime-self.startTime)修改成print(sys.stderr, ‘\nTimeElapsed: %s‘ % (self.stopTime-self.startTime))

有问题加群QQ×××流: 610845268

